packages/browser/src/plugins/schema-filter/__tests__/index.test.ts

```diff
@@ -472,7 +472,7 @@ describe('schema filter', () => {
       expect(updateUserProfile.track).toHaveBeenCalled()
     })
 
-    it('covers different names between remote plugins and integrations', async () => {
+    it('works when current name differs from creation name', async () => {
```

Comment:
Can we add a test that would have failed without this fix? 2 integrations where the 2nd one's name starts with the entirety of the 1st one?